test-integration/helpers/utils/wait.coffee

```diff
@@ -4,8 +4,11 @@ class Wait
   constructor: (test) -> @test = test
 
   # TODO reduce the copy pasta between for and forMultiple
-  forMultiple: (locator, ms = 60 * 1000) ->
+  forMultiple: (locator, ms = 60 * 1000) =>
     locator = @test.utils.toLocator(locator)
+    locator.shouldBeVisible ?= true
```

Comment:
Was there discussion on this somewhere else (maybe in a diff that was closed)? I thought the conclusion was to not add fields to the locator, no?